niworkflows/conftest.py

```diff
@@ -29,12 +29,23 @@
 import pandas as pd
 import pytest
 import tempfile
-import pkg_resources
+
+try:
+    import importlib_resources
+except ImportError:
+    import importlib.resources as importlib_resources
```

Comment:
`importlib.resources` is present I think in all supported versions, so this would prevent us from using the backported features.

An alternative is to import the specific required functions:

```Python
try:
    from importlib.resources import files, as_file
except ImportError:
    from importlib_resources import files, as_file
```

But they're a little cryptic without the module name. We could do `import files as ...` but then we'd need to come up with a name.